module-db/Tables/MultimediaFilesTable.cpp

```diff
@@ -8,6 +8,20 @@
 #include <magic_enum.hpp>
 #include <inttypes.h>
 
+namespace
+{
+    std::string getSorting(db::multimedia_files::SortingBy sorting)
+    {
+        switch (sorting) {
+        case db::multimedia_files::SortingBy::Id:
+            return "_id ASC";
+        case db::multimedia_files::SortingBy::Title:
```

Comment:
What about:
```suggestion
        case db::multimedia_files::SortingBy::Title:
        default:
```
That way there will be no need to repeat exactly the same string in return

Reply:
done